diff --git a/spec/ruby/core/file/utime_spec.rb b/spec/ruby/core/file/utime_spec.rb
index 542681e..64adcf4 100644
--- a/spec/ruby/core/file/utime_spec.rb
+++ b/spec/ruby/core/file/utime_spec.rb
@@ -42,14 +42,14 @@
     File.mtime(@file2).to_i.should be_close(@mtime.to_i, 2)
   end
 
-  platform_is :linux do
-    platform_is wordsize: 64 do
-      it "allows Time instances in the far future to set mtime and atime" do
-        time = Time.at(1<<44)
-        File.utime(time, time, @file1)
-        File.atime(@file1).year.should == 559444
-        File.mtime(@file1).year.should == 559444
-      end
-    end
-  end
+  # platform_is :linux do
+  #   platform_is wordsize: 64 do
+  #     it "allows Time instances in the far future to set mtime and atime" do
+  #       time = Time.at(1<<44)
+  #       File.utime(time, time, @file1)
+  #       File.atime(@file1).year.should == 559444
+  #       File.mtime(@file1).year.should == 559444
+  #     end
+  #   end
+  # end
 end
