--- ruby-3.0.4/test/zlib/test_zlib.rb	2022-04-12 07:48:55.000000000 -0400
+++ ruby-3.0.3/test/zlib/test_zlib.rb	2022-09-21 18:51:25.706677885 -0400
@@ -788,26 +788,6 @@
         gz.close
       }
     end
-
-    if defined? File::TMPFILE
-      def test_path_tmpfile
-        sio = StringIO.new("".dup, 'w')
-        gz = Zlib::GzipWriter.new(sio)
-        gz.write "hi"
-        gz.close
-
-        File.open(Dir.mktmpdir, File::RDWR | File::TMPFILE) do |io|
-          io.write sio.string
-          io.rewind
-
-          gz = Zlib::GzipWriter.new(io)
-          assert_raise(NoMethodError) { gz.path }
-
-          gz = Zlib::GzipReader.new(io)
-          assert_raise(NoMethodError) { gz.path }
-        end
-      end
-    end
   end
 
   class TestZlibGzipReader < Test::Unit::TestCase
